src/QueryEditor/Space.tsx

```diff
@@ -10,6 +10,9 @@ export interface SpaceProps {
   layout?: 'block' | 'inline';
 }
 
+/**
+ * @deprecated use the Space component from @grafana/ui instead. Available starting from @grafana/ui@10.4.0
```

Comment:
10.4 is not released yet, but I've added this here to show that an alternative is available.